contracts/registries/LicenseRegistry.sol

```diff
@@ -19,12 +19,18 @@ import { DataUniqueness } from "../lib/DataUniqueness.sol";
 contract LicenseRegistry is ILicenseRegistry, ERC1155, Governable {
     using Strings for *;
 
-    /// @dev Name of the License NFT
+    /// @notice Emitted for metadata updates, per EIP-4906
+    event BatchMetadataUpdate(uint256 _fromTokenId, uint256 _toTokenId);
+
+    /// @notice Name of the License NFT
     string public name = "Story Protocol License NFT";
```

Comment:
please update to "Programmable IP License NFT" to be consistent with the branding.